### PR TITLE
[REVIEW] Add docstring for `dask_cudf.read_csv`

### PR DESCRIPTION
**Pull request by galipremsagar**
Fixes: #2277 

This PR adds python docstring for `dask_cudf.read_csv` API.

### CODE REVIEW

File: python/dask_cudf/dask_cudf/io/csv.py [hunk 1]
```diff
@@ -16,6 +16,52 @@
 
 
 def read_csv(path, chunksize="256 MiB", **kwargs):
+    """
+    Read CSV files into a Dask_cudf.DataFrame
```
Comment (kkraus14) [inline python/dask_cudf/dask_cudf/io/csv.py:20]:
```suggestion
    Read CSV files into a dask_cudf.DataFrame
```

File: python/dask_cudf/dask_cudf/io/csv.py [hunk 2]
```diff
@@ -16,6 +16,52 @@
...
+    >>> import dask_cudf
+    >>> df = dask_cudf.read_csv("myfiles.*.csv")
+
+    In some cases it can break up large files:
+
+    >>> df = dask_cudf.read_csv("largefile.csv", chunksize="256 MiB")
+
+    It can read CSV files from external resources (e.g. S3, HTTP, FTP)
+
+    >>> df = dask_cudf.read_csv('s3://bucket/myfiles.*.csv')
```
Comment (kkraus14) [inline python/dask_cudf/dask_cudf/io/csv.py:35]:
```suggestion
    >>> df = dask_cudf.read_csv("s3://bucket/myfiles.*.csv")
```

**Review by kkraus14 — APPROVED** [review]

File: python/dask_cudf/dask_cudf/io/csv.py [hunk 3]
```diff
@@ -16,6 +16,52 @@
...
+
+    Parameters
+    ----------
+    path : str, path object, or file-like object
+        Either a path to a file (a str, pathlib.Path, or
+        py._path.local.LocalPath), URL (including http, ftp, and S3 locations),
+        or any object with a read() method (such as builtin open() file
+        handler function or StringIO).
+    chunksize : int or str, default "256 MiB"
+        The target task partition size.
```
Comment (ayushdg) [inline python/dask_cudf/dask_cudf/io/csv.py:51]:
```suggestion
        The target task partition size. If `None`, a single block is used for each file.
```

File: python/dask_cudf/dask_cudf/io/csv.py [hunk 4]
```diff
@@ -16,6 +16,52 @@
...
+    **kwargs : dict
+        Passthrough key-word arguments that are sent to ``cudf.read_csv``.
+
+    Examples
+    --------
+    >>> import dask_cudf
+    >>> ddf = dask_cudf.read_csv('sample.csv', usecols=['a', 'b'])
```
Comment (ayushdg) [inline python/dask_cudf/dask_cudf/io/csv.py:58]:
```suggestion
    >>> ddf = dask_cudf.read_csv("sample.csv", usecols=["a", "b"])
```

**Review by ayushdg — APPROVED** [review]

**Comment by kkraus14** [comment]
rerun tests

**Comment by galipremsagar** [comment]
rerun tests

**Comment by galipremsagar** [comment]
@gpucibot merge

**Pull request merged**